docs/bookmarks.md

```diff
@@ -218,4 +218,13 @@ To resolve a conflicted state in a remote bookmark (e.g. `main@origin`), simply
 pull from the remote (e.g. `jj git fetch`). The conflict resolution will also
 propagate to the local bookmark (which was presumably also conflicted).
 
+## Ease of use
+
+The use of bookmarks is frequent in some workflows, for example, when
+interacting with git repositories containing branches. To this end,
```

Comment:
nit: I think we try to write "Git" when talking about the tool (and `git` when talking about the command).

Reply:
Fixed